### ISSUE TITLE
Does not work with ts-jsx-loader

### ISSUE THREAD

**Issue by Ciantic** [s-panferov/awesome-typescript-loader#10]
I'm using this: https://github.com/jbrantly/ts-jsx-loader because it is the only valid way (to keep IDE support) to deal with JSX in TypeScript, but unfortunately I can't get it working with normal way:

**webpack.config.js**

```
module.exports = {
    resolve: {
        extensions: ['', '.ts', '.js']
    },
    devtool: 'source-map',
    module: {
        loaders: [{
            test: /\.ts$/,
            loader: 'awesome-typescript-loader!ts-jsx-loader'
        }]
    },
    entry: {
        index: ['./index.ts']
    },
    output: {
        path: '../',
        filename: './[name].js'
    }
};
```

It's as if ts-jsx-loader is never called. And the React.jsx entries are not transformed.

**Comment by s-panferov** [comment]
@Ciantic I know what is wrong here. I'll try to fix this soon.

**Comment by s-panferov** [comment]
@Ciantic 	landed in v0.3.0
